app/com/linkedin/drelephant/tuning/PSOParamGenerator.java

```diff
@@ -80,11 +85,12 @@ public JobTuningInfo generateParamSet(JobTuningInfo jobTuningInfo) {
     String jobType = jobTuningInfo.getJobType().toString();
 
     List<String> error = new ArrayList<String>();
-
+    logger.info("String State "+ stringTunerState);
```

Comment:
These logging statements are not needed. These are long strings which do no help in any way and just clutter the log file.